ui/components/NotificationsTable.tsx

```diff
@@ -100,7 +101,7 @@ function NotificationsTable({ className, rows }: Props) {
     return (
       <Flex wide tall column align>
         <Spacer padding="xxl" />
-        <MessageFlex column align={false}>
+        <MessageBox column align={false}>
```

Comment:
align={false} can be deleted I think since I was overriding the default props. If no value is provided align-items will be `start`